JavaScript/JavaScriptSDK/Initialization.ts

```diff
@@ -130,6 +130,7 @@ module Microsoft.ApplicationInsights {
             config.verboseLogging = Util.stringToBoolOrDefault(config.verboseLogging);
             config.emitLineDelimitedJson = Util.stringToBoolOrDefault(config.emitLineDelimitedJson);
             config.diagnosticLogInterval = config.diagnosticLogInterval || 10000;
+            config.samplingPercentage = config.samplingPercentage || 100;
```

Comment:
if customer sets sampling percentage 0, they get 100, because 0||100 is 100 is this intentional?